presto-native-execution/presto_cpp/main/operators/PartitionAndSerialize.h

```diff
@@ -25,7 +25,7 @@ class PartitionAndSerializeNode : public velox::core::PlanNode {
   PartitionAndSerializeNode(
       const velox::core::PlanNodeId& id,
       std::vector<velox::core::TypedExprPtr> keys,
-      int numPartitions,
+      int32_t numPartitions,
```

Comment:
We should really have this as unsigned int. :)

Reply:
Yeah, agreed. Changed. Talked with @oerling that we might want to use int32 instead of int. For numPartitions kind of use cases, shall we use vector_size_t or uint32_t.